docs/src/main/paradox/r2dbc.md

```diff
@@ -57,6 +57,12 @@ The table below shows `akka-projection-r2dbc`'s direct dependencies, and the sec
 The `akka_projection_offset_store`, `akka_projection_timestamp_offset_store` and `akka_projection_management` tables
 need to be created in the configured database:
 
+@@@ warning
```

Comment:
@patriknw As it is now, this warning is shown regardless of the chosen dialect tab. 

So, in a-p-r2dbc, we took advantage of the `paradoxGroups` which allowed to make this a [condition](https://github.com/akka/akka-persistence-r2dbc/blob/main/docs/src/main/paradox/migration.md?plain=1#L62-L66).

To get this warning only render if "SQLServer" tab is checked, we would also have to introduce `paradoxGroups` in the `build.sbt`, however this would show in all pages (that have no reference to those dialects). Any idea? I tried to hide the html-rendering of the dialect drop-down in the top-left via JS, and it worked, but it was not perfect.

Reply:
That's fine, let's show the warning for all dialects here.